src/components/propTypesTodoShape/propTypeUserShape.js

```diff
@@ -0,0 +1,9 @@
+import PropTypes from 'prop-types';
+
+const propTypeUserShape = {
+  user: PropTypes.shape({
+    name: PropTypes.string.isRequired,
+  }).isRequired,
+};
```

Comment:
1. С большой буквы
2. Некорректно, в переменную записывай результат PropTypes.shape
3. Удали слово propType из названия переменной

```suggestion
const propTypeUserShape = PropTypes.shape({
    name: PropTypes.string.isRequired,
  }).isRequired,
```

Reply:
Сколько ошибок, спасибо большое за вевью, уже пушу с исправлениями